docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db.md

```diff
@@ -111,20 +116,98 @@ Camunda document reference metadata, similarity score, and the actual text conte
 
 ## Embedding models
 
-### Amazon Bedrock
+<Tabs groupId="embedding-models" defaultValue="bedrock" queryString values={
+[
+{label: 'Amazon Bedrock', value: 'bedrock' },
+{label: 'Azure OpenAI', value: 'azure-openai' },
+{label: 'Google Vertex AI', value: 'vertex-ai' },
+{label: 'OpenAI', value: 'openai' }
+]}>
 
+<TabItem value='bedrock'>
 The **vector database connector** currently supports only [Amazon Titan V1/V2 models](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html).
 Review the [official Amazon documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan-embed-text.html) to understand how to choose request parameters.
 
 The **vector database connector** uses [LangChain4j implementation](https://docs.langchain4j.dev/integrations/embedding-models/amazon-bedrock).
```

Comment:
Not sure how much value we have by keeping this paragraph and if we keep it we should probably add the same for the other models. I'd suggest to remove it.